src/vivarium/config_tree.py

```diff
@@ -513,6 +513,10 @@ def _set_with_metadata(
 
     def __setattr__(self, name, value):
         """Set a value on the outermost layer."""
+        # https://stackoverflow.com/a/50888571/
+        if name.startswith("__") and name.endswith("__"):
```

Comment:
Shouldn't this be inside the `if name not in self:` block?

Reply:
Hm... I didn't fully think this through. My initial idea was that it should be consistent between getting and setting. The way I've written it here, we allow ConfigTree to hold keys that look like dunders, but they can't be get/set with dot syntax, only with bracket syntax.

If you'd prefer that they can be get/set with dot syntax _once they exist_ but can only be _created_ with bracket syntax (I think this is how pandas columns work), I could do what you suggest, and make a change to the `__getattr__` as well to make that work if the key already exists.

There is probably some argument for not allowing them at all, but that seems like a bigger change.